app/src/main/res/layout/search_menu.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
```

Comment:
Could you address the Cirrus warnings please? You can define the strings in the string.xml file. For the warning about the buttons I am not sure what they mean.